src/components/dom.js

```diff
@@ -98,4 +159,126 @@ export default class Dom {
 
     }
 
+    /**
+     * Checks target if it is native input
+     * @param {Element|*} target - HTML element or string
+     * @return {boolean}
+     */
+    static isNativeInput(target) {
+
+        let nativeInputs = [
+            'INPUT',
+            'TEXTAREA'
+        ];
+
+        return target ? nativeInputs.indexOf(target.tagName) !== -1 : false;
+
+    }
+
+    /**
+     * Checks node if it is empty
+     * It must be node without childNodes
+     * @param {Node} node
+     *
+     * @return {Boolean} true if it is empty
+     */
+    static checkNodeEmpty(node) {
+
+        let nodeText;
+
+        if ( this.isElement(node) && this.isNativeInput(node) ) {
+
+            nodeText = node.value;
+
+            if ( nodeText.trim() ) {
```

Comment:
You can move this block under the 
```js 
if ( this.isElement(node) && this.isNativeInput(node) ) {
} else {
}
``` 
and remove similar one from the second part

Reply:
And replace it with 
```js
return nodeText.trim().length === 0;
```